genesis.go

```diff
@@ -1,8 +1,12 @@
 package hotstuff
 
+import "time"
+
 var genesisBlock = NewBlock(Hash{}, QuorumCert{}, "", 0, 0)
 
 // GetGenesis returns a pointer to the genesis block, the starting point for the hotstuff blockchain.
 func GetGenesis() *Block {
+	ts := time.Unix(0, 0).UTC() // should we use a static string for parsing?
```

Comment:
I think we should do the timestamp initialization either via `NewBlock` or in an `init()` function so that we don't incur any construction cost when calling `GetGenesis`, even though we rarely call it.

We could probably use a timestamp newer than the Unix zero value (1970) time... Something like 2025 Jan 1, but it is not important, I think.